app/controllers/users_controller.rb

```diff
@@ -0,0 +1,14 @@
+class UsersController < ApplicationController
+
+  def index
```

Comment:
I don't think that there should be an index route in this controller. The root should have the login button, but I am not sure that this should be associated with a user. Maybe we can have a controller just for the root, and then when a user logs out, redirect back to the root?